src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.builds

```diff
@@ -5,6 +5,9 @@
     <Project Include="System.Security.Cryptography.Algorithms.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.Security.Cryptography.Algorithms.csproj" Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
@bartonjs Do you know what the earliest version of .NET Framework this library can target? Currently it we are defaulting it to .NET 4.6.